index/server/tests/Expectations/base64_index.json

```diff
@@ -0,0 +1,73 @@
+[
```

Comment:
nit: dir should start with lowercase? upto you

Reply:
Sure, I rename those test folders.